src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  text-align: left;
+  color: #060B35;
 }
+
+.card {
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  width: 198px;
+  height: 406px;
+  position: relative;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  text-align: center;
+}
+
+  .card__image {
+    background-image: url('../images/imac.jpeg');
+    background-size: cover;
+    background-position: center;
+    width: 160px;
+    position: absolute;
+      height: 134px;
+      top: 33px;
+      left: 20px;
+  }
+
+  .card__content {
+ padding: 20px 15px;
+  }
+
+ .card__title {
+   font-size: 12px;
+   font-weight: 500;
+   line-height: 1.5;
+   margin: 0 0 8px;
+   color: #060B35;
+    }
+  .card__product {
+    font-size: 10px;
+    font-weight: 400;
+    color:#060B35;
+   }
+
+  .card__rating-wrapper {
+    display: flex;
+    justify-content: center;
+    gap: 5px;
+    margin: 12px 0;
+}
+
+.card__star svg {
+  fill: #e5e5e5;
+  cursor: pointer;
+}
+
+.card__star:hover svg,
+.card__star:hover~.card__star svg {
+  background-color: #FFDE6A;
+  stroke-width: 0;
+}
+
+.card__reviews {
+  font-size: 10px;
+  font-weight: 400;
+  color: #fff;
```

Comment:
The color for `.card__reviews` is set to white. Ensure that the background where this text appears is not white, or change the text color to ensure it is visible.